.travis.yml

```diff
@@ -44,13 +44,14 @@ matrix:
         - os: osx
           env: PYTHON_VERSION=2.7 SETUP_CMD='test'
                CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
+               PIP_DEPENDENCIES='jplephem'
```

Comment:
I would recommend making a new variable higher up called ``PIP_ALL_DEPENDENCIES`` and then using that whenever ``CONDA_ALL_DEPENDENCIES`` is also used.